project/build.properties

```diff
@@ -1 +1,2 @@
 sbt.version=0.13.15
+scalaVersion := "2.12.3"
```

Comment:
@moonpatroller This shouldn't be needed.  It is already in build.sbt as `scalaVersion := "2.12.3"`
I'd rather not have it here and let sbt use whatever version in needs.